<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_VAV_terminal</class>
 <widget class="QDialog" name="Dialog_VAV_terminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>389</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>671</width>
    <height>389</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>671</width>
    <height>389</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog_VAV_terminal
{
background-color:white;
}

QLineEdit 
{
font-family: 'Microsoft YaHei';
font-size: 13px;
}

QComboBox
{
font-family: 'Microsoft YaHei';
font-size: 13px;
}</string>
  </property>
  <widget class="QWidget" name="widget_top" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_top
{
border: 1px solid #000000;
background-color: rgb(157, 196, 229);
	background-color: rgb(157, 196, 229);
}</string>
   </property>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/image/closeWindow.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_VAV_terminal" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>651</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_VAV_terminal
{
border: 1px solid #000000;
}</string>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>611</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_2
{
border: 1px solid #DCDCDC;
}

QLabel
{
font-family: 'Microsoft YaHei';
font-size: 16px;
}</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>90</y>
       <width>581</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>63Hz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_63">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>125Hz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_125">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>250Hz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_250">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>500Hz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_500">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1kHz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_1k">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>2kHz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2k">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>4kHz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_4k">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>8kHz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_8k">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="minimumSize">
           <size>
            <width>76</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>总值dB(A)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_total">
          <property name="minimumSize">
           <size>
            <width>76</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>阀门开度</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_valve_open_degree">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>型号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>品牌</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_number">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>编号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_model">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_brand">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_air_volume">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>风量(m³/h)</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_confirm">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color:#ffffff; /*文字颜色*/
    background-color: #00ACB9;/*背景色*/
    border-radius:0px; /*边框倒角*/
	border: 1px solid #000000;
}


QPushButton:hover 
{
background-color: rgb(0,142,153);
color: white;
}</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>45</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_18">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { 
border: none; 
background-color : rgb(105, 105, 105);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>24</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { 
border: none; 
color : rgb(105, 105, 105);
}</string>
       </property>
       <property name="text">
        <string>变风量末端</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
